packages/analytics/amplify_analytics_pinpoint_dart/lib/src/impl/analytics_client/endpoint_client/endpoint_store_keys.dart

```diff
@@ -5,6 +5,15 @@
 enum EndpointStoreKey {
   /// The current version of the endpoint version store
   version,
+
+  /// Global attributes key attached to endpoint
+  endpointGlobalAttributesKey,
```

Comment:
```suggestion
  globalAttributesKey,
```

Reply:
While it makes sense in code to drop the endpoint, I think it's important to actually keep an endpoint prefix in case we store more values into the secure storage and need a way to easily differentiate keys.

Reply:
I switched the version to endpointStoreVersion as well to be consistent with the above logic

Reply:
This is already `EndpointStoreKey` - reiterating that these values are related to endpoints is not necessary. If you're concerned about the value in storage (and not in code), then you can override the `name` used for storage:

```dart
enum EndpointStoreKey {
  version('endpointStoreVersion'),

  ...;

  const EndpointStoreKey(this.name);

  final String name;
}
```

But even this should not be necessary since values should already be appropriately scoped to the endpoint store, making this, too, redundant. Overall, I'm not seeing what values you would be adding here which would need further designation as endpoint values.